spark/src/main/scala/io/spark/ddf/etl/TransformationHandler.scala

```diff
@@ -96,26 +101,20 @@ class TransformationHandler(mDDF: DDF) extends CoreTransformationHandler(mDDF) {
     val rMapped = dfrdd.map {
```

Comment:
@ctn you are right. The eval call eventually call silentlyEval which in some cases returns null, but it's a null of type REXP. I guess Scala just auto-cast whatever null into it's Nothing type?